.gitignore

```
@@ -1,2 +1,5 @@
 .deploy
-.idea
\ No newline at end of file
+.idea
+
+./invenio/charts/*
```

```suggestion
invenio/charts/
```
This should fix git reporting the charts pulled by `helm dependency update` as untracked